### ISSUE TITLE
Modification du formulaire

### ISSUE THREAD

**Issue by mlhamel** [mlhamel/agendadulibre#20]
- Supprimer la portée
- Remanier le champs de tags afin de proposer des tags reliés à la description 
- Rendre le courriel personne qui soumet facultatif et ajouter une mention comme quoi cela est facultatif
- Ameliorer le captcha, par exemple on pourrait transformer le captcha en question mathematiques
- Enlever le champs Annoncé
- Rendre certains champs comme facultatif (Nom de l'endroit, tags, Adresse)

**Comment by millette** [comment]
Pourquoi avoir supprimé la porté? C'était pratique pour la SQIL, par exemple, et d'autres événements/journées nationales et internationales.

Certains événements avaient lieu à Ottawa (tout près, quand même), maintenant, on ne peut plus les ajouter sans tricher vu qu'il y a la liste de 1000 muni du Québec exclusivement.

**Comment by mlhamel** [comment]
Effectivement, mais le formulaire était beaucoup trop compliqué et dans 95% des cas personne ne semblait l'utiliser.... Est-ce qu'on devrait vraiment le remettre tu crois ?

**Comment by millette** [comment]
Y'a plusieurs journées internationnales. J'ajouterais simplement les options à la liste des muni:
* Ottawa
* Canada
* Provinciale
* Internationale

Je crois que ce serait le plus simple.

**Comment by mlhamel** [comment]
+1, est-ce que voudrais crée un ticket avec cette idée ?
